test/models/competitor_test.rb

```diff
@@ -244,10 +244,10 @@ class CompetitorTest < ActiveSupport::TestCase
     end
   end
 
-  test '#birthday_on?' do
+  test '#birthday_on_competition?' do
```

Comment:
instead of changing the existing test, can you just add a new one (and leave the old one as it was)?